geometry_msgs/Pose goal
---
string message
